spec/puppet-server-lib/puppet/jvm/http_client_spec.rb

```diff
@@ -21,4 +21,19 @@
       expect(request_options.get_connect_timeout_milliseconds).to equal(connect_timeout)
     end
   end
+
+  context "requests" do
```

Comment:
More conventional to use "describe" here since I get the impression you're describing requests.

I typically use context when the spec is expressed in the "given ... when ... then" format.

They're aliases of each other so this purely semantic.